### PR TITLE
README and code comment fixes

### PR DESCRIPTION
**Pull request by stefanionescu**
Fixes for main READMEs and package files

### CODE REVIEW

File: packages/astroport/src/vesting.rs [hunk 1]
```diff
@@ -5,150 +5,150 @@ use cosmwasm_std::{Addr, Order, Uint128};
...
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 #[serde(rename_all = "snake_case")]
 pub enum QueryMsg {
     /// ## Description
-    /// Returns information about the vesting configs in the [`ConfigResponse`] object.
+    /// Returns the configuration for the contract using a [`ConfigResponse`] object.
     Config {},
     /// ## Description
-    /// Returns information about the vesting account in the [`VestingAccountResponse`] object.
+    /// Returns information about an address vesting tokens using a [`VestingAccountResponse`] object.
```
[the remark this answers is not in the document]
Comment (stefanionescu) [reply packages/astroport/src/vesting.rs:102]: Address being vested and vesting is kinda the same thing no?

File: packages/astroport/src/vesting.rs [hunk 2]
```diff
@@ -5,150 +5,150 @@ use cosmwasm_std::{Addr, Order, Uint128};
...
     VestingAccount {
         address: String,
     },
     /// ## Description
-    /// Returns a list of accounts, for the given input parameters, in the [`VestingAccountsResponse`] object.
+    /// Returns a list of addresses that are vesting tokens using a [`VestingAccountsResponse`] object.
```
[the remark this answers is not in the document]
Comment (stefanionescu) [reply packages/astroport/src/vesting.rs:107]: Same here, these are accounts that are vesting tokens so can use vesting no?

File: packages/astroport/src/vesting.rs [hunk 3]
```diff
@@ -5,150 +5,150 @@ use cosmwasm_std::{Addr, Order, Uint128};
...
     pub info: VestingInfo,
 }
 
 /// ## Description
-/// This structure describes the custom struct for each query response.
+/// This structure describes a custom struct used to return vesting data for multipel vesting targets.
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 pub struct VestingAccountsResponse {
-    /// the vesting accounts information
+    /// A list of accounts that are vesting tokens
```
[the remark this answers is not in the document]
Comment (stefanionescu) [reply packages/astroport/src/vesting.rs:146]: They didn't finish vesting, they are still vesting here right? So can use "vesting tokens"

File: packages/astroport/src/vesting.rs [hunk 4]
```diff
@@ -183,7 +182,7 @@ pub struct MigrateMsg {}
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 #[serde(rename_all = "snake_case")]
 pub enum Cw20HookMsg {
-    /// Register vesting accounts
+    /// RegisterVestingAccounts registers vesting targets/accounts
```
[the remark this answers is not in the document]
Comment (stefanionescu) [reply packages/astroport/src/vesting.rs:185]: The accounts didn't finish vesting so can say "vesting accounts" no?

File: packages/astroport/src/vesting.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (ShadoySV) [reply packages/astroport/src/vesting.rs:102]: Yes, sorry

**Review by ShadoySV — APPROVED** [review]

**Pull request merged**